src/Illuminate/Foundation/Console/StorageLinkCommand.php

```diff
@@ -11,7 +11,8 @@ class StorageLinkCommand extends Command
      *
      * @var string
      */
-    protected $signature = 'storage:link {--relative : Create the symbolic link using relative paths}';
+    protected $signature = 'storage:link {--relative : Create the symbolic link using relative paths}
+                {--force : Recreate already existing symbolic links}';
```

Comment:
```suggestion
    protected $signature = 'storage:link
                {--relative : Create the symbolic link using relative paths}
                {--force : Recreate already existing symbolic links}';
```

Reply:
This makes more sense to me too, originally I chose the other format to be consistent with other commands (I looked at `migrate`)

Reply:
Seems like it's a bit all over the place and we hold different ways of doing this. I'll put it on my list to consolidate at some point. Thanks for pointing that out.